package.json

```diff
@@ -21,7 +21,7 @@
     "focus-visible": "^5.2.0",
     "gray-matter": "^4.0.3",
     "javascript-time-ago": "^2.5.9",
-    "next": "13.3.0",
+    "next": "^14.2.14",
```

Comment:
please remove any upgrades, your pr should not be changing this

Reply:
Is it ok now?

Reply:
I still see some unwanted changes on the other files.
I think the best way to fix all the extra files that are not needed is to copy paste your one liner change in a notepad, then do a reset hard of the whole branch to main, and then readd your fix and commit only this file with `git add src/components/NavLink.jsx`

Reply:
- @violetadev , Give it a check nowww?!